flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/calls/ScalarOperatorGens.scala

```diff
@@ -1287,12 +1287,16 @@ object ScalarOperatorGens {
            |boolean $nullTerm;
            |if (${condition.resultTerm}) {
            |  ${trueAction.code}
-           |  $resultTerm = ${trueAction.resultTerm};
+           |  if (!${trueAction.nullTerm}) {
```

Comment:
This will lead to a risk, if `trueAction.nullTerm` is a expression instead of a variable, maybe will some problem, so I suggest you:
```
$nullTerm = ${trueAction.nullTerm};
if (!$nullTerm) {
  $resultTerm = ${trueAction.resultTerm};
}
```
At least, we can ensure `nullTerm` is just a variable.

Reply:
sure, thanks for the suggestion. 
code updated.

Reply:
BTW, I encountered below error running `ScalarOperatorsTest` after updating the code:

```
java.lang.VerifyError: Expecting a stackmap frame at branch target 32
Exception Details:
  Location:
    TestFunction$163.map(Ljava/lang/Object;)Ljava/lang/Object; @19: ifne
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0x0000000: 2bc0 000e 4d2c 100d b900 1402 0036 0401
    0x0000010: 4e15 049a 000d 2c10 0d05 b900 1803 004e
    0x0000020: 2d3a 4415 049a 000e 2ab4 001c 1944 b600
    0x0000030: 223a 442c 05b9 0014 0200 3632 0236 3115
......
```

It seems to be a bug in Janino. After I searched Janino's issues, I found this https://github.com/janino-compiler/janino/issues/90#issuecomment-494025582

Have you ever met the same problem?

Reply:
Never encounter... But maybe because our code generate code has some problem, we can run again after fixing.

Reply:
@libenchao @JingsongLi 

Hello! This might be a bit unrelated, but I am developing on flink and I am also running into a very similar `VerifyError` pointing to a Janino code generated method (specifically in the Flink protobuf row serialization codepath). How did you workaround/resolve this issue?

Reply:
@kurnoolsaketh We didn't solve it, it just disappeared.

Reply:
interesting.. thanks for the info.